packages/dds/tree/src/test/domains/json/jsonCursor.bench.ts

```diff
@@ -112,6 +138,38 @@ const canada = generateCanada(
         ? undefined
         : [2, 10]);
 
-describe("ITreeCursor", () => {
-    bench("canada", () => canada);
+function* extractCoordinatesFromCanada(cursor: ITreeCursor): Generator<[number, number]> {
```

Comment:
in my experience generators can be a bit slow. You might want to compare this to an inlined version and/or a visitor pattern (take a callback to pass the x and y to, instead of allocating an array and yielding that for each one.

Reply:
The visitor version is a bit slower than the inlined version (and the generator version much slower than both) but I decided to go with the visitor version to make it easier to provide consumers for different datasets.